.github/workflows/reusable-add-reviewers-to-pr.yml

```
@@ -0,0 +1,53 @@
+name: Add reviewers to PRs
+
+on:
+  pull_request:
+    types: ['labeled']
+  pull_request_target:
+    branches: ['master', 'next']
```

Shouldnt it be like this? as we talked on the other repo?

```suggestion
  pull_request_target:
    branches: ['master', 'next']
    types: ['labeled']
```

---

it does not really make a difference... Will add the change though!